README.md

```diff
@@ -21,120 +21,101 @@ If you want to deploy your swarm immediately you can use our docker-compose quic
 5. Create a node server application using our node.js [library](https://github.com/bluzelle/bluzelle-js) and [API](https://bluzelle.github.io/api/)
 6. `CTRL-C` to terminate the docker-compose swarm
 
-## Getting started building from source
+## Building from source
 
-### Installation - macOSX
+### Dependencies - Minimum version requirements. Boost 1.67.0, gcc 7.1, cmake 3.12 and Protocol buffers 3.6.0.
+Note, appropriate versions are in some cases provided by the system's package manager (apt, brew, yum, ect). In other cases, you might need to update your sources.lists to a test branch, or install the dependecies from source to obatin a supported version. 
 
-**Boost**
+### Linux Ubuntu 18.04
```

Comment:
Please create separate install steps for 18.04. No need to deprecate all other builds.